fiaas_skipper/deploy/cluster.py

```diff
@@ -31,6 +32,12 @@ def find_deployment_configs(name, namespace=None):
         res = []
         configmaps = ConfigMap.find(name, namespace)
         for c in configmaps:
-            tag = c.data['tag'] if 'tag' in c.data else 'stable'
-            res.append(DeploymentConfig(name=name, namespace=c.metadata.namespace, tag=tag))
+            tag = c.data.get('tag', 'stable')
+            sa_per_app = c.data.get('enable-service-account-per-app', False)
```

Comment:
On second thought I think this will not entirely work either; In my previous comment I made a mistake and forgot that the actual fiaas-deploy-daemon configuration is not contained directly under the `.data` key (sorry about that). The configuration is actually set as a string containing yaml under the `.data.cluster_config.yaml` key in the configmap (see the helm template [here](https://github.com/fiaas/skipper/blob/master/helm/fiaas-skipper/templates/fiaas-deploy-daemon-configmap.yaml)). 
Because of this, looking up the value of the `enable-service-account-per-app` config flag will be a bit more complicated than this. I think it would be a good idea to extend the tests for find_deployment_configs [here](https://github.com/fiaas/skipper/blob/bootstrap-with-sa-per-app/tests/fiaas_skipper/deploy/test_cluster.py) to include test data with this config flag set/unset (and with/without other config flags) and then make the implementation here pass the new/updated tests

Reply:
good point, I missed that one. Tests are now added.